dexlib2/src/main/java/org/jf/dexlib2/writer/DexWriter.java

```diff
@@ -250,19 +270,28 @@ private int getDataSectionOffset() {
     }
 
     /**
-     * Checks whether any of the size-sensitive constant pools have overflowed.
-     *
-     * This checks whether the type, method, field pools are larger than 64k entries.
+     * Checks whether any of the size-sensitive constant pools has overflowed and has more than 64Ki entries.
      *
      * Note that even if this returns true, it may still be possible to successfully write the dex file, if the
-     * overflowed items are not referenced anywhere that uses a 16-bit index
+     * overflowed items are not referenced anywhere that uses a 16-bit index.
      *
-     * @return true if any of the size-sensitive constant pools have overflowed
+     * @return true if any of the size-sensitive constant pools has overflowed
```

Comment:
The subject is plural ("any of the pools"), so "have" is correct. Same for the similar sentence in the method description above.

Reply:
ok, will fix. i'm not a native english speaker, the plural just sounded strange to me. "any answer will do" and other similar expressions (as well as usage in my language) made me feel like "any pool is" and "any of the pools is" were the same thing. :+1:

Reply:
yeah, "any pool" is singular, but "any of the pools" is plural. Why? Heck if I know. lol :)

Reply:
5 instances changed from HAS to HAVE